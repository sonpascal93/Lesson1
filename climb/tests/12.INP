2000
10 -39 22 -8 15 47 83 97 -37 73 -55 -54 -33 5 -83 -34 51 83 -81 98 -46 -24 14 -71 -49 -45 -24 1 33 -86 0 6 12 -93 -5 57 63 -56 -31 -19 52 30 -22 -30 -46 -90 80 -57 -92 14 23 74 -34 -90 12 -8 -40 -63 -61 72 13 37 32 -80 -10 -5 -97 24 39 -15 96 88 -21 -57 21 -84 58 50 -94 -23 27 -32 -88 -44 11 26 91 39 -81 -27 -78 46 54 29 -76 -61 -24 -63 25 -93 28 92 -37 -66 -93 -34 76 -29 19 79 12 59 65 -79 31 -20 88 56 70 25 -9 -41 -91 62 59 80 12 4 94 0 -33 64 65 -3 94 3 33 25 -95 -28 24 87 -6 41 -4 -85 53 58 89 -89 -13 -5 18 -29 64 40 52 -29 36 -42 -5 67 -82 22 -67 -45 -96 62 -94 10 2 -62 -90 -6 65 -66 -55 89 7 -67 58 -80 -41 99 47 96 14 84 6 61 -29 -29 -28 -32 50 -98 -4 37 -17 -27 39 54 -2 -57 -79 10 -60 41 -14 20 30 74 92 -5 -21 83 -88 -97 -71 -83 75 78 -54 67 51 48 31 -44 -26 75 83 95 -21 -4 22 -27 -73 -97 -11 62 93 -52 73 -52 30 52 1 -46 17 4 86 8 -1 -70 -91 -8 -43 13 -4 -61 4 44 70 12 70 -24 -98 3 -3 -78 90 -73 21 -47 -8 27 0 66 17 26 91 -5 56 -79 -39 72 36 -86 27 -81 98 63 54 -42 36 -12 -4 -59 91 29 -50 81 -53 -66 -77 -80 52 -41 15 2 -86 -18 -17 -41 -31 1 -94 5 70 23 -3 51 19 20 -13 -51 -59 -10 22 76 -71 -2 44 51 73 -54 -21 43 -44 85 -97 68 -43 55 -55 -31 41 28 89 -35 -88 86 -95 -44 -33 -80 64 71 -62 -3 41 51 -32 0 -67 83 -1 64 -69 -81 0 36 -92 88 -4 99 -46 -83 -36 -98 64 -88 70 -95 -71 90 83 77 11 -84 89 -74 -59 -52 -78 21 82 -89 46 5 14 85 92 -84 27 3 69 52 81 -85 48 98 -7 -95 80 50 20 -11 98 61 -70 -56 -64 51 94 23 -67 -45 -38 -29 -26 -39 -35 -31 5 71 -79 36 -59 -41 99 42 -91 1 -90 -81 -31 40 45 75 -52 -67 -78 70 0 -18 -45 -29 -73 98 -97 51 47 76 28 -47 9 -24 -6 63 36 71 -95 -77 99 -25 -6 13 -54 -32 -35 -96 -43 -43 -53 58 52 -7 28 85 87 62 -77 -80 32 11 -37 42 90 -83 54 -40 -22 63 42 94 -19 -44 90 -10 -61 78 -24 -12 83 -25 29 68 -64 -31 -12 -19 13 32 -5 -31 -17 -52 -95 -74 -84 -41 -15 -76 -17 26 98 -49 -84 -77 -66 -79 80 -46 -34 74 -62 94 94 -1 61 89 76 80 11 -88 -66 96 -27 -96 9 -89 77 73 99 -93 3 83 -38 -15 8 97 75 -37 28 25 36 14 -62 85 -46 -72 -53 -29 -56 -31 -86 59 -86 56 -28 -86 -11 11 -39 -3 0 -33 -82 -7 -86 -79 57 22 -4 22 0 -82 50 52 -74 -82 39 81 -98 84 0 -52 -97 -23 -83 38 14 14 -13 50 57 -7 61 -56 -5 -20 -18 -81 14 -58 -61 53 8 69 -13 -57 -46 18 -8 -9 -7 -33 9 7 16 -78 -20 -86 53 60 76 32 -31 78 -42 32 73 28 91 -9 -44 28 66 66 16 -22 79 -44 -70 75 -5 -23 -37 -8 -98 -76 -68 72 -46 -96 67 -17 47 0 82 -23 87 -87 68 -90 12 36 -93 33 -59 -17 -5 -26 18 -49 91 -55 -63 71 -98 57 67 24 -92 95 55 -13 -30 27 86 -14 -81 -34 89 -28 60 -11 -41 -17 -63 -63 97 -75 -38 84 -86 -24 4 41 -37 21 -48 -12 0 -98 44 65 -65 -31 -93 20 -73 83 12 19 -59 48 64 13 87 24 -1 -88 -87 91 -59 -16 85 -70 53 -34 -59 85 -99 -22 62 26 0 -9 39 -64 64 -40 -5 -34 11 38 89 17 48 -38 37 -45 -40 -63 81 -5 0 55 -1 70 66 49 -49 0 -73 50 -3 -25 16 -7 58 72 64 -45 25 43 99 0 -55 -97 65 73 -2 -53 -26 -25 -56 1 42 -55 41 -60 88 22 -76 -45 48 12 58 90 65 -88 -84 -33 95 -49 -97 2 78 28 54 91 46 -95 -41 -3 -61 28 99 -84 -80 18 -89 -25 46 33 -23 88 15 73 40 69 6 99 -35 9 -83 43 66 56 -34 67 93 28 43 -90 -59 -35 -92 -78 82 41 26 -95 18 -31 84 86 -94 -78 -50 79 -50 -58 43 68 -6 -79 -64 39 -26 -28 -71 71 0 -78 -24 -32 50 -1 83 -94 -29 -40 12 91 -15 64 98 62 5 45 13 64 73 37 69 -75 13 71 -92 -30 91 30 39 86 -84 67 -38 -12 21 78 19 -49 44 1 -51 56 -48 -92 -65 -96 -60 -7 13 8 -72 -39 22 -16 53 -30 73 95 -56 22 -44 35 -39 70 -87 -71 -28 -17 -31 29 52 97 -38 -60 73 65 -12 4 44 63 -9 77 0 43 -28 -4 -5 51 -62 26 34 45 -37 -95 43 75 90 -95 83 26 99 -40 -24 -79 -21 23 26 53 -51 -87 -24 -30 -41 17 -87 -45 59 34 69 78 -63 -35 19 25 54 -88 55 -59 44 -26 75 -25 23 69 -84 83 96 -11 96 -39 11 27 -53 71 -6 -95 19 94 -5 86 25 -39 -33 -83 -57 40 16 -92 84 45 -2 -87 -11 -55 -52 -53 -37 97 44 78 -18 -91 34 26 -65 70 74 -91 -59 -53 8 25 -6 -82 -6 -12 43 -50 58 -98 -93 -41 97 51 -51 -71 17 -14 -5 -58 -60 -59 -80 -93 -82 67 -46 27 43 -38 -60 -34 23 26 -55 -32 8 16 58 -29 0 14 -21 -3 -48 8 -36 -77 -2 -22 -94 -85 24 74 87 14 -34 83 -31 22 84 14 52 37 -26 21 -40 36 43 43 -96 -39 -10 -1 -67 8 56 -64 57 -61 35 -33 -96 -82 0 -91 -26 5 -83 57 4 24 94 21 -53 -6 37 -22 41 23 97 -15 -35 87 8 -98 -36 -43 -35 42 21 32 -69 -49 -45 -2 -1 -63 -65 34 7 24 2 -12 12 -6 62 74 1 93 23 -35 44 78 -38 1 -99 -97 -58 52 55 82 -23 -77 35 60 -35 -47 -90 93 60 85 68 67 76 22 -29 66 -63 -88 -37 11 65 70 71 27 18 -58 -66 74 -8 -91 -67 -41 32 0 -45 68 37 -39 -47 -97 -48 27 77 19 -26 -35 60 84 89 76 -89 -7 99 71 82 -38 69 81 82 22 -14 1 -3 -26 10 -57 -35 64 7 -81 -68 79 83 11 -95 30 -72 81 -58 -56 -55 -87 -46 -93 5 -66 34 -40 -5 50 29 48 -84 -73 16 -57 -64 -21 18 -49 18 -26 34 -60 -67 37 -64 -35 83 46 69 -66 29 52 47 34 -16 -36 29 -8 85 -21 13 -44 15 15 -5 10 -86 -47 61 -13 -56 76 -8 39 24 90 50 82 -9 30 -16 10 91 83 84 -73 11 -39 -25 70 -26 -7 -85 -18 14 83 -22 74 -16 -7 44 58 -99 21 2 -4 -20 -72 61 -7 -88 11 11 -37 -78 -44 -97 -6 -22 57 -26 70 18 -99 -49 30 -9 -54 67 56 79 -89 -99 -75 99 60 -80 56 13 -1 91 -59 36 91 -10 -46 5 83 74 81 29 50 -18 -37 -11 -73 90 21 76 -14 25 12 18 -34 -11 -65 30 -50 23 13 38 76 -86 5 87 -53 -58 -49 -36 29 -60 -59 65 28 -43 45 33 -21 -79 50 76 -10 95 14 -89 -29 -74 18 82 -25 -26 -13 -83 -86 73 61 63 -61 14 -15 23 -65 -74 54 9 89 -52 56 -94 -55 -68 -26 -52 27 -29 -87 1 -46 -75 5 -58 7 69 39 -85 -94 -96 28 -63 -31 18 -55 -99 -51 -92 -28 25 76 53 -50 -24 -68 25 39 64 42 -27 2 45 -73 -75 78 18 -99 -48 -64 -62 88 51 -56 -7 32 31 69 46 72 71 -29 61 -85 -99 -17 1 -35 38 -99 57 80 73 51 -28 51 91 -77 -31 86 -12 2 57 -68 15 -79 96 75 -95 37 -31 -55 -17 -15 82 61 -96 -3 -24 49 76 65 21 -80 36 6 -77 -67 -56 -23 55 -7 -56 -91 51 50 -83 81 -81 -96 -68 59 -3 -80 -20 4 3 -31 98 67 -34 -52 -58 78 -2 -67 31 -56 -65 93 -17 19 28 13 -60 18 -17 -39 31 -60 -98 76 61 -48 63 47 -56 55 52 75 77 89 29 38 -89 -28 27 15 -27 76 50 -85 75 -21 -74 -43 -52 -84 44 -18 -55 -2 99 81 -99 -71 -92 0 -79 -16 -45 -53 -55 -94 88 -50 51 93 -5 82 74 -54 75 -16 8 65 31 14 49 -40 67 -87 32 6 85 -61 -89 40 25 -4 -99 -16 -47 78 2 61 39 -48 -18 -64 -63 7 -74 -98 -28 -41 -18 -40 -57 32 -78 89 -6 98 -25 88 42 -48 27 -56 -79 74 3 -65 5 -76 41 82 -26 41 -73 49 44 -83 42 97 21 -64 52 49 -55 -16 87 18 34 -20 10 16 5 -56 -85 97 84 6 -67 97 -83 -29 -44 -1 -82 -88 52 -9 -43 -16 14 90 43 -73 11 -48 -49 70 -72 28 -70 -36 -47 -7 -91 87 91 -97 -58 -13 47 56 -61 97 64 -15 82 -65 -44 15 33 -32 -7 1 -69 77 -73 73 11 46 -46 -5 50 -14 -17 -28 69 56 -54 72 0 94 -16 4 72 58 7 -30 -90 32 16 -95 -80 -70 56 -70 -91 -63 72 13 -98 -81 -72 -84 -30 -7 79 27 92 -27 -33 -16 95 61 -33 87 95 6 -95 44 50 34 -20 -57 -55 -54 9 -54 -82 35 -91 39 -49 -30 -30 -85 78 81 -45 -41 95 84 -36 47 -73 -10 70 68 62 -88 -32 33 -27 93 -28 94 -22 -93 -34 -24 69 49 95 71 52 -93 -84 8 -51 22 76 -84 -95 28 -23 96 20 57 -52 27 -21 -64 40 -23 96 -24 -92 -45 89 -78 -52 -4 91 -46 30 81 -16 -26 47 42 -84 -29 -82 58 94 -38 -3 43 -67 67 -18 61 -69 47 -57 92 