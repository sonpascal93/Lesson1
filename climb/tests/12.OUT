129
34 49 54 83 89 91 95 98 103 106 131 154 158 191 192 193 200 229 233 239 256 275 282 297 316 318 323 324 329 333 334 344 357 368 381 383 388 397 402 403 412 419 441 460 499 509 515 543 554 563 564 663 684 687 713 748 759 777 800 804 839 855 867 880 887 894 912 938 946 988 1008 1018 1025 1042 1051 1065 1075 1136 1140 1154 1160 1174 1186 1189 1221 1227 1234 1274 1303 1329 1367 1394 1402 1412 1429 1447 1457 1462 1490 1502 1503 1524 1527 1542 1597 1609 1646 1649 1666 1694 1709 1741 1746 1766 1768 1798 1833 1837 1842 1849 1851 1877 1886 1888 1920 1934 1942 1950 1957 